lutris/gui/lutriswindow.py

```diff
@@ -439,8 +443,14 @@ def _open_browser(url):
 
     @GtkTemplate.Callback
     def on_resize(self, widget, *args):
-        """WTF is this doing?"""
-        self.window_size = widget.get_size()
+        """size-allocate signal
```

Comment:
Docstrings should use proper grammar. Ie., this should be capitalised and use punctuation: "Size-allocate signal."